src/Controls/src/Core/ScrollView/ScrollView.cs

```diff
@@ -31,7 +28,7 @@ public Rect LayoutAreaOverride
 					return;
 				_layoutAreaOverride = value;
 				// Dont invalidate here, we can relayout immediately since this only impacts our innards
-				UpdateChildrenLayout();
+				InvalidateMeasure();
```

Comment:
I see this one is already obsolete in .NET9: can we remove the property completely?

```
[Obsolete]
public Rect LayoutAreaOverride
```

Reply:
We can't remove it completely because it's part of `IScrollViewController` :-( 

So, for now we can just change this to an explicit implementation of `IScrollViewController` and at least get it removed from the `txt` files